### PR TITLE
Playbook and guide for creating windows custom VM

### PR DESCRIPTION
**Pull request by StefanStojanovic**
This PR adds a new playbook, based on `jenkins/worker/create.yml`, and a guide on how to use it to set up a custom Windows VM for building and testing Node.js.

This PR also removes python 2 from Windows machines, since it is no longer used, plus it comes up in the `PATH` variable before python 3.

Refs: https://github.com/nodejs/build/issues/3137

### CODE REVIEW

**Review by richardlau — APPROVED** [review]

File: doc/create-windows-custom-vm.md
Comment (StefanStojanovic) [reply doc/create-windows-custom-vm.md:89]: OK, I'll remove it as a follow on PR after this lands. Speaking of landing it, what is the policy on landing PRs in this repo? Should I wait for 2 days or not?

> shouldn't they be "ICU4C"?

the tasks are actually named ICU2C, but the URLs they use, point to ICU4C

Comment (richardlau) [reply doc/create-windows-custom-vm.md:89]: I think 2 days is a reasonable time before landing PRs here. The main thing is getting reviews/approvals (at least one, you have one from me).

**Pull request merged**
